content/04.results.md

```diff
@@ -61,18 +65,27 @@ Note: for Orivet, the conditions were not balanced because, due to experimenter
+
+### MOST TESTS ACCURATELY REFLECTED PUREBRED DOGS’ REGISTERED BREEDS
+
 For all dogs, the registered breed was always the majority estimate across tests (Figure @fig:pies; Supplementary Table 4).
-Most tests returned at least one result that did not match the dog's breed organization registration at 100%.
-The primary outcome selected *a priori* (Table @tbl:accuracy) did not capture apparent qualitative differences (Figure @fig:pies).
-This discrepancy was driven by the fact that, in several cases, the breeds predicted by a test matched neither the DNA nor the photograph.
-This pattern was particularly apparent for the image classifier and one DTC genetic test (ViaPet by Accu-Metrics).
-In the case of the neural network, this was likely influenced by the limited number of breeds in the training data, and thus category labels, in ImageNet [@url:http://vision.stanford.edu/aditya86/ImageNetDogs].
-For example, ImageNet is not trained to identify the Chinese Crested and Bulldog breeds.
-In the case of the genetic test, the results may have been influenced by the fact that the test predicted the registered breed zero times across both the control and treatment conditions (Table @tbl:accuracy).
-Re-evaluating the results at the level of breed clades (Figure @fig:tern) revealed that, in both cases, the predicted breeds were more similar to the photographs provided than to the DNA sample analyzed.
-While the effects did not meet the level of statistical significance (Supplementary Table 5), the odds ratios indicate that the treatment condition had a similar effect on the ability of NASNet and Accu-Metrics to predict the breed of the DNA donor (odds ratio of 13 versus 11).
+However, most tests did not match every dog with its registered breed at 100%.
+This primary outcome, selected *a priori* (Table @tbl:accuracy), therefore did not capture apparent qualitative differences in breed predictions (Figure @fig:pies).
+In several cases, the breeds predicted by a test matched neither the DNA nor the photograph.
+This pattern was particularly apparent for both the image classifier and one DTC genetic test (ViaPet by Accu-Metrics).
+In the case of this genetic test, this pattern may have been influenced by the fact that the test predicted the registered breed zero times across both the control and treatment conditions (Table @tbl:accuracy).
+
+Re-evaluating the results at the level of breed clades (Figure @fig:tern) suggested potential explanations for results that matched the breed of neither the DNA donor nor the photograph.
+In the cases of both NASNet and Accu-Metrics, the effects did not meet the level of statistical significance (Supplementary Table 5), but the odds ratios suggest that the condition of swapped photographs affected the predictions of both approaches.
+The odds that these tests predicted a breed in the same clade as the DNA donor were 13 for NASNet and 11 for Accu-Metrics (Supplementary Table 5).
```

Comment:
```suggestion
The odds ratios for a breed in the same clade as the DNA donor were 13 for NASNet and 11 for Accu-Metrics, but these results are only suggestive as they did not meet our predefined threshold for significance (Supplementary Table 5).
```
CHECK THIS - ARE YOU SURE THIS IS DNA donor and not photograph? I may misunderstand this result.

Reply:
I think this is phrased to match the table below-- it's the odds ratio of the DNA donor's clade being in the results. I think I did it this way because it's very close to being the _a priori_ primary outcome we selected, just zoomed out to the clade level.

<img width="671" alt="Screenshot 2023-12-30 at 7 19 41 AM" src="https://github.com/rando2/dtcdog-ms/assets/8975841/ec508021-60d2-4771-a9f9-62f5d421415f">

```suggestion
When the tests were evaluated based on whether they predicted any breed in the same clade as the DNA donor at any percentage, NASNet performed with 100% accuracy in the control condition and 0% accuracy in the treatment condition, corresponding to an odds ratio of 13 (Supplementary Table 5).
Accu-Metrics identified breeds in the same clade as the donor twice in the control condition and zero times in the treatment condition, corresponding to an odds ratio of 11 (Supplementary Table 5).
However, these results are only suggestive, as neither met our predefined threshold for significance.
```